src/machine/machine_rp2040_pwm.go

```diff
@@ -180,6 +194,12 @@ func (p *pwmGroup) SetTop(top uint32) {
 	p.setWrap(uint16(top))
 }
 
+// SetCounter sets counter control register. Max value is 16bit (0xffff).
+// Useful for synchronising two different PWM peripherals.
+func (p *pwmGroup) SetCounter(ctr uint32) {
+	p.CTR.Set(ctr)
+}
+
```

Comment:
I guess this is analogous to the `Counter()` method?
I guess this is fine. I mostly added `Counter()` for debugging purposes though.

Reply:
The counter method has been used by someone to sync PWM signals. I'd leave this.